block_additional_mappings/smpte12m_timecode.md

```diff
@@ -0,0 +1,51 @@
+## Block Additional Mapping - SMTPE 12M Timecode
+
+### Timecode Description
+
+SMPTE 12 timecode values can be stored in the `BlockMore Element` to associate the content of a Matroska Block with a particular timecode value.
+
+The Block Additional Mapping contains a full binary representation of a 64 bit SMPTE timecode value expressed exactly as defined in Section 7 and 8 of SMPTE 12M [@!SMPTE.12M.2014]. For convenience, here are the bit assignments from SMPTE 12M, for information:
```

Comment:
Are we legally allowed to copy parts of this spec ?

Reply:
I am copying parts of https://tools.ietf.org/html/rfc5484#section-6.2, but perhaps I should clarify that the text for convenience is derived from RFC5484 rather than a proprietary SMPTE document. ping to @mcr on this issue.